### PR TITLE
Feature/CON-33/passage scrolling support new ui

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/CON-33
Related to: https://github.com/oat-sa/tao-test-runner-qti-fe/pull/335

Fixed issue with Save - update `container.body` after Active Include widget exit
Added support for new UI test runner - added classes `tao-overflow-y` for height `tao-full-height` and other

How to test:

1. create passage
2. create item with this passage
3. enable scroll for passage and save item
4. go to test Preview and check that passage is scrollable

### CODE REVIEW

File: views/js/qtiCreator/widgets/static/helpers/itemScrollingMethods.js [hunk 1]
```diff
@@ -22,59 +22,100 @@ define([
...
             isScroll ? $form.find('.scrollingSelect').show() : $form.find('.scrollingSelect').hide();
             height && $form.find('.scrollingSelect select').val(height).change();
         },
         wrapContent: function (widget, value, wrapType) {
             const $form = widget.$form;
-            let $wrap = wrapType === 'inner' ? widget.$container
+            let $wrapper = wrapType === 'inner' ? widget.$container
                 .find('[data-html-editable]')
-                .children('.custom-text-box') : widget.$container.parent('.custom-text-box');
+                .children(`.${wrapperTextCls}`) : widget.$container.parent(`.${wrapperIncludeCls}`);
```
Comment (andrey-ponamarev) [inline views/js/qtiCreator/widgets/static/helpers/itemScrollingMethods.js:73]: I believe `parent` should be replaced with `closest`
```suggestion
                .children(`.${wrapperTextCls}`) : widget.$container.closest(`.${wrapperIncludeCls}`);
```

File: views/js/qtiCreator/widgets/static/text/states/Sleep.js [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+                    container.body(newBody);
+                }
+            }
+        }, 'question');
+    }, function(){
+        this.widget.offEvents('question');
+    });
     
     return TextBlockStateSleep;
 });
```
Comment (habemuscode) [inline views/js/qtiCreator/widgets/static/text/states/Sleep.js:27]: Empty line at the final of file

**Review by habemuscode — APPROVED** [review]
One small comment but approved, check code climate, at least the error avoid size limit of function. The other two, maybe we cannot fix for don't create dependency

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

File: views/js/qtiCreator/widgets/static/helpers/itemScrollingMethods.js [hunk 3]
```diff
@@ -22,59 +22,100 @@ define([
...
+        setScrollingHeight: function ($wrapper, value) {
+            $wrapper.attr('data-scrolling-height', value);
+            const opt = options.find((opt) => opt.value === value);
+            $wrapper.addClass(opt.class);
         },
-        setScrollingHeight: function (wrapper, value) {
-            wrapper.attr('data-scrolling-height', value);
+        cutScrollClasses: function (classes) {
+            let clearClasses = classes.replace(wrapperFocusCls, '');
+            clearClasses = clearClasses.replace(newUIclass, '');
```
[the remark this answers is not in the document]
Comment (bziondik) [reply views/js/qtiCreator/widgets/static/helpers/itemScrollingMethods.js]: fixed

**Review by andrey-ponamarev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Pull request merged**